ooler_mqtt_bridge

```diff
@@ -114,6 +71,18 @@ async def control_cleaning(mqtt: Client, myooler: ooler.Ooler) -> None:
             await send_update(mqtt, myooler)
 
 
+async def control_device_units(mqtt: Client, myooler: ooler.Ooler) -> None:
+    """Watch for and act on instructions to change the device units"""
+    message_filter = f"ooler/{sanitise_mac(myooler.address)}/temp_units/set"
+    async with mqtt.filtered_messages(message_filter) as messages:
+        async for message in messages:
+            payload = message.payload.decode()
+            new_units = constants.TemperatureUnit[payload]
+            await myooler.set_temperature_unit(new_units)
+            await send_discovery(mqtt, myooler)
```

Comment:
This is "overkill", but I don't know of another way to change the device payload, other than to simply resend it. Since HA models a `climate` device as having a static temperature unit, this is the only way I could find to update the units in HA on-the-fly.

Reply:
Seems reasonable to me, I imagine it's unlikely to be flooding the MQTT broker/consumer